astropy/units/__init__.py

```diff
@@ -54,6 +54,9 @@
 __all__ += structured.__all__
 __all__ += equivalencies.__all__
 __all__ += function.__all__
+# We sort `__all__` for the docs. We use `globals` to avoid confusing any static
+# analysis tools.
+globals()["__all__"] = sorted(__all__)
```

Comment:
Why not just `globals()["__all__"].sort()`? `__all__` is guaranteed to be a `list` and that has an in-place `sort`.

Reply:
Can do. Just personal preference re in-place mutations.

Reply:
Done!